REGRESSION(263115@main): [CMake][WPE][GTK] Distcheck broken

aperezdc opened the pull request:
#### 14502b3258b8c74727a7c41bfea0fef780b8fc70
<pre>
REGRESSION(263115@main): [CMake][WPE][GTK] Distcheck broken
<a href="https://bugs.webkit.org/show_bug.cgi?id=255746">https://bugs.webkit.org/show_bug.cgi?id=255746</a>

Reviewed by Michael Catanzaro and Myles C. Maxfield.

* Source/WebCore/Headers.cmake: Move out Cocoa-specific headers out from
  the share CMake fragment.
* Source/WebCore/PlatformMac.cmake: Move said headers here, into the
  port-specific CMake fragment.

Canonical link: <a href="https://commits.webkit.org/263210@main">https://commits.webkit.org/263210@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8580cd3b46d2630661c8e7f1c529ba866810be4b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/3913 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/4003 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/4114 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/5349 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/4171 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/3890 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/4112 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/3996 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/3983 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/3961 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/4170 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/3532 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/5204 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/1672 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/3507 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/5545 "") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/3493 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/3567 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/4975 "17 api tests failed or timed out") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/3966 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/3203 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/3486 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/3507 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/3532 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/451 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/3764 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

litherum reviewed (approved):
Whoops! Thanks for fixing this!!!

aperezdc committed:
REGRESSION(263115@main): [CMake][WPE][GTK] Distcheck broken
https://bugs.webkit.org/show_bug.cgi?id=255746

Reviewed by Michael Catanzaro and Myles C. Maxfield.

* Source/WebCore/Headers.cmake: Move out Cocoa-specific headers out from
  the share CMake fragment.
* Source/WebCore/PlatformMac.cmake: Move said headers here, into the
  port-specific CMake fragment.

Canonical link: https://commits.webkit.org/263210@main
